package.json

```diff
@@ -42,37 +42,37 @@
     "prepare": "is-ci || husky install"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^5.5.0"
+    "@typescript-eslint/experimental-utils": "^5.9.0"
   },
   "devDependencies": {
-    "@babel/eslint-plugin": "^7.14.5",
+    "@babel/eslint-plugin": "^7.16.5",
     "@commitlint/cli": "^16.0.1",
     "@commitlint/config-conventional": "^16.0.0",
-    "@types/jest": "^27.0.3",
-    "@types/node": "^16.11.11",
-    "@typescript-eslint/eslint-plugin": "^5.5.0",
-    "@typescript-eslint/parser": "^5.5.0",
+    "@types/jest": "^27.4.0",
+    "@types/node": "^16.11.19",
+    "@typescript-eslint/eslint-plugin": "^5.9.0",
+    "@typescript-eslint/parser": "^5.9.0",
```

Comment:
Since `@typescript-eslint/experimental-utils` shouldn't be upgraded, these two either for consistency within the ecosystem.